### ISSUE TITLE
Inputs causing canvas to lose focus

### ISSUE THREAD

**Issue by lifzgreat** [michaeldzjap/react-signature-pad-wrapper#7]
On mobile, when an input has focus (and the keyboard is hidden so you can continue down the page), and you draw on the signature pad, when you release the touch, focus jumps back to the input. If the input is much higher on the page, it scrolls back up the page to the input and pops the keyboard back up. It's not necessarily a bug, just the current HTML behavior; but it's certainly nonintuitive for a user.

In order to fix this HTML behavior, one needs to handle `onMouseDown` and/or `onTouchStart` in the canvas. For example:
```javascript
<SignaturePad
    ref={ref => this.signaturePad = ref}
    canvasProps={{
        onMouseDown: () => { if ('activeElement' in document) document.activeElement.blur(); },
        onTouchStart: () => { if ('activeElement' in document) document.activeElement.blur(); },
    }}
/>
```

However, in order to do _that_, we need to be able to pass props directly to your wrapper's `canvas` element (in `render()`):
```javascript
<canvas
    className={this.props.className}
    ref={ref => this._canvas = ref} {...this.props.canvasProps} />
```

Would you mind giving us the ability to pass props directly to the canvas?

**Comment by michaeldzjap** [comment]
Sure, that seems like a reasonable request to me.

**Comment by michaeldzjap** [comment]
Just published [v1.2.0](https://github.com/michaeldzjap/react-signature-pad-wrapper/tree/v1.2.0) which should solve this issue
